src/auth.js

```diff
@@ -142,22 +142,41 @@ export default class DropboxAuth {
   }
 
   generatePKCECodes() {
-    let codeVerifier = crypto.randomBytes(PKCELength);
-    codeVerifier = codeVerifier.toString('base64')
-      .replace(/\+/g, '-')
-      .replace(/\//g, '_')
-      .replace(/=/g, '')
-      .substr(0, 128);
+    let codeVerifier;
+    function convertToStringAndReplace(toBeConverted) {
+      const convertedString = toBeConverted.toString('base64')
+        .replace(/\+/g, '-')
+        .replace(/\//g, '_')
+        .replace(/=/g, '');
+      return convertedString;
+    }
+
+    if (isBrowserEnv()) {
+      const array = new Uint32Array(10);
+      crypto.getRandomValue(array)
+        .then((randomValueArray) => {
```

Comment:
Check method definition for this: https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues